mappings/net/minecraft/entity/player/PlayerEntity.mapping

```diff
@@ -1,46 +1,114 @@
 CLASS net/minecraft/class_54 net/minecraft/entity/player/PlayerEntity
 	FIELD field_506 playerCapeUrl Ljava/lang/String;
-	FIELD field_508 spawnPos Lnet/minecraft/class_63;
+	FIELD field_507 sleepOffsetY F
+	FIELD field_508 sleepingPos Lnet/minecraft/class_63;
+	FIELD field_509 sleepOffsetX F
+	FIELD field_510 sleepOffsetZ F
+	FIELD field_511 portalCooldown I
+	FIELD field_512 inTeleportationState Z
 	FIELD field_514 fishHook Lnet/minecraft/class_135;
 	FIELD field_515 sleepTimer I
+	FIELD field_516 spawnPos Lnet/minecraft/class_63;
+	FIELD field_517 ridingStartPos Lnet/minecraft/class_63;
+		COMMENT Used for the "On A Rail" Achievement
 	FIELD field_519 inventory Lnet/minecraft/class_136;
-	FIELD field_520 playerContainer Lnet/minecraft/class_71;
-	FIELD field_521 container Lnet/minecraft/class_71;
+	FIELD field_520 playerScreenHandler Lnet/minecraft/class_71;
+	FIELD field_521 currentScreenHandler Lnet/minecraft/class_71;
+	FIELD field_522 unused B
```

Comment:
This byte in named decompiled code is userType.
In a previous version its only usage was here
(mojmap)
```
private void handleMouseClick(int paramInt) {
    if (paramInt == 0 && this.missTime > 0)
      return; 
    if (paramInt == 0)
      this.player.swing(); 
    boolean bool = true;
    if (this.hitResult == null) {
      if (paramInt == 0 && !(this.gameMode instanceof net.minecraft.client.gamemode.CreativeMode))
        this.missTime = 10; 
    } else if (this.hitResult.type == HitResult.Type.ENTITY) {
      if (paramInt == 0)
        this.gameMode.attack((Player)this.player, this.hitResult.entity); 
      if (paramInt == 1)
        this.gameMode.interact((Player)this.player, this.hitResult.entity); 
    } else if (this.hitResult.type == HitResult.Type.TILE) {
      int i = this.hitResult.x;
      int j = this.hitResult.y;
      int k = this.hitResult.z;
      int m = this.hitResult.f;
      Tile tile = Tile.tiles[this.level.getTile(i, j, k)];
      if (paramInt == 0) {
        this.level.extinguishFire(i, j, k, this.hitResult.f);
        if (tile != Tile.unbreakable || this.player.userType >= 100)
          this.gameMode.startDestroyBlock(i, j, k, this.hitResult.f); 
      } else {
        ItemInstance itemInstance = this.player.inventory.getSelected();
        boolean bool1 = (itemInstance != null) ? itemInstance.count : false;
        if (this.gameMode.useItemOn((Player)this.player, this.level, itemInstance, i, j, k, m)) {
          bool = false;
          this.player.swing();
        } 
        if (itemInstance == null)
          return; 
        if (itemInstance.count == 0) {
          this.player.inventory.items[this.player.inventory.selected] = null;
        } else if (itemInstance.count != bool1) {
          this.gameRenderer.itemInHandRenderer.itemPlaced();
        } 
      } 
    } 
    if (bool && paramInt == 1) {
      ItemInstance itemInstance = this.player.inventory.getSelected();
      if (itemInstance != null && this.gameMode.useItem((Player)this.player, this.level, itemInstance))
        this.gameRenderer.itemInHandRenderer.itemUsed(); 
    } 
  }```

In the b1.7.3 version under biny it is missing:
```
    private void method_2107(int i) {
        ItemStack class_312;
        if (i == 0 && this.field_2787 > 0) {
            return;
        }
        if (i == 0) {
            this.player.swingHand();
        }
        boolean bl = true;
        if (this.hitResult == null) {
            if (i == 0 && !(this.interactionManager instanceof TestInteractionManager)) {
                this.field_2787 = 10;
            }
        } else if (this.hitResult.type == HitResultType.ENTITY) {
            if (i == 0) {
                this.interactionManager.attackEntity(this.player, this.hitResult.entity);
            }
            if (i == 1) {
                this.interactionManager.interactEntity(this.player, this.hitResult.entity);
            }
        } else if (this.hitResult.type == HitResultType.BLOCK) {
            int n = this.hitResult.blockX;
            int n2 = this.hitResult.blockY;
            int n3 = this.hitResult.blockZ;
            int n4 = this.hitResult.side;
            if (i == 0) {
                this.interactionManager.attackBlock(n, n2, n3, this.hitResult.side);
            } else {
                int n5;
                ItemStack class_313 = this.player.inventory.getSelectedItem();
                int n6 = n5 = class_313 != null ? class_313.count : 0;
                if (this.interactionManager.interactBlock(this.player, this.world, class_313, n, n2, n3, n4)) {
                    bl = false;
                    this.player.swingHand();
                }
                if (class_313 == null) {
                    return;
                }
                if (class_313.count == 0) {
                    this.player.inventory.main[this.player.inventory.selectedSlot] = null;
                } else if (class_313.count != n5) {
                    this.field_2818.field_2342.method_1863();
                }
            }
        }
        if (bl && i == 1 && (class_312 = this.player.inventory.getSelectedItem()) != null && this.interactionManager.interactItem(this.player, this.world, class_312)) {
            this.field_2818.field_2342.method_1865();
        }
    }
    ```

I can only assume it was something to do with privileges? just considering that it allows you to break an unbreakable tile, maybe the name `privilege` because it isnt used in yarn any where else and clearly its been removed from the game, or like `unusedPrivilege`
I dunno but could be better than `unused`

Reply:
Also looked at the b1.2 leak, but i decided to leave it just as `unused` to not imply any function to that variable

Reply:
fair enough!